.circleci/config.yml

```diff
@@ -922,13 +922,19 @@ jobs:
 
   deploy-to-kubernetes:
     description: "Deploy Grafana master Docker image to Kubernetes"
+    parameters:
+      edition:
+        type: string
     executor: base
     steps:
+      - attach_workspace:
+          at: /tmp/workspace
       - install-grabpl
       - run:
           name: Deploy to Kubernetes
           command: |
-            /tmp/grabpl deploy-to-k8s $CIRCLE_WORKFLOW_ID
+            cp -r /tmp/workspace/<< parameters.edition >>/dist .
```

Comment:
Can we remove the parameter and just hardcode enterprise, since it doesn't vary anyway? I prefer not introducing parameters unless it's necessary, since it makes things easier to verify.